Homework4/Task1.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+
+
+def functions_results(func):
+    """Stores the results of function calls and the time it takes to get
+
+    the result during program start
+    """
+    if 'results' not in globals():
+        global results
+        results = {}
+
+    def wrapper(*args, **kwargs):
+        before_call = datetime.datetime.now()
+        res = func(*args, **kwargs)
+        after_call = datetime.datetime.now()
+        time = after_call - before_call
```

Comment:
as I remember it would be datetime.timedelta. it would need conversion and would return formatted time. overall it's ok. alternatively you can use from time import time